root/tree/tree/h5hep/examples/Makefile

```diff
@@ -0,0 +1,13 @@
+CFLAGS=-std=c++17 -O2 -Wall -I..
```

Comment:
I am wondering why we use Make here rather than CMake?

Reply:
This was an internal, independent, side project for the evaluation we carried out for ACAT 2021; that's why.  It would be useful to migrate to CMake and add some tests.

Reply:
Can we do this as part of this PR? I think it shouldn't be very hard.